src/components/tcping/QvTCPing.hpp

```diff
@@ -0,0 +1,32 @@
+#pragma once
+#include "base/Qv2rayBase.hpp"
+
+namespace Qv2ray::components::tcping
+{
+    struct QvTCPingData {
+        ConnectionIdentifier connectionIdentifier;
+        QString hostName;
+        int port;
+        QString errorMessage;
+        int total, succeed, failed;
+        double min = 999999999999999.0, max = 0.0, avg = 0.0;
```

Comment:
import `<limits>` and use `std::numeric_limits<double>::max()` to replace this ugly thing.

Reply:
ack

Reply:
It was initially ported from C, which was what the original author did.
We’ll make good use of c++17

Reply:
This is already fixed.